zerver/lib/mention.py

```diff
@@ -272,39 +278,40 @@ def init_user_group_data(self, realm_id: int, content: str) -> None:
         self.user_group_name_info: dict[str, NamedUserGroup] = {}
         self.user_group_members: dict[int, set[int]] = defaultdict(set)
         user_group_names = possible_user_group_mentions(content)
+        notified_user_group_names = non_silent_user_group_mentions(content)
```

Comment:
`possibly_notifiable_user_group_names` would be more correct. We've not verified it's actually a mention yet, just that potentially relevant syntax appears (but it could be in a code block or something).

Reply:
Strongly agree !